test/Conversion/TorchToTosa/selective_decomposition.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: torch-mlir-opt <%s -convert-torch-backend-legal-to-tosa-custom="custom-ops=torch.aten._softmax" -split-input-file -verify-diagnostics | FileCheck %s
+
+// CHECK-LABEL:   func.func @torch.aten._softmax(
+// CHECK-SAME:      %[[VAL_0:.*]]: !torch.vtensor<[4,4],f32>) -> !torch.vtensor<[4,4],f32> {
+// CHECK:           %[[VAL_1:.*]] = torch_c.to_builtin_tensor %[[VAL_0]] : !torch.vtensor<[4,4],f32> -> tensor<4x4xf32>
+// CHECK:           %[[VAL_2:.*]] = torch.constant.int -1
+// CHECK:           %[[VAL_3:.*]] = torch.constant.bool false
+// CHECK:           %[[VAL_4:.*]] = "tosa.const"() {value = dense<-1> : tensor<i64>} : () -> tensor<i64>
+// CHECK:           %[[VAL_5:.*]] = "tosa.const"() {value = dense<0> : tensor<i64>} : () -> tensor<i64>
+// CHECK:           %[[VAL_6:.*]] = "tosa.custom"(%[[VAL_1]], %[[VAL_4]], %[[VAL_5]]) {identifier = "torch.aten._softmax"} : (tensor<4x4xf32>, tensor<i64>, tensor<i64>) -> tensor<4x4xf32>
```

Comment:
A small nit here: let's have the "identifier" omit the "torch." but have the "config" added in https://reviews.llvm.org/D137133 be set to `torch_mlir`.

Reply:
Sure! I can change it, though the current LLVM version in Torch-MLIR does not have the commit you shared ([link to TOSA CustomOp arguments](https://github.com/llvm/llvm-project/blob/74fb770de9399d7258a8eda974c93610cfde698e/mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td#L1798)).
I can update the `identifier` to omit the "torch." part for now and we can either wait for an LLVM uplift to bring in 'config' or keep having it marked as TODO. Either way makes sense to me.

Reply:
I updated the 'identifier' attribute of the TOSA CustomOp, also added _a_ `config` attribute. Please note that TOSA CustomOps in this version of LLVM still do not have the `config` attribute, so it's a monkey-patch right now. Added a TODO to update the builder call after the LLVM uplift.

Reply:
replace with this should work.
`// CHECK:           %[[VAL_6:.*]] = "tosa.custom"(%[[VAL_1]], %[[VAL_4]], %[[VAL_5]]) {config = "torch_mlir", identifier = "aten._softmax"} : (tensor<4x4xf32>, tensor<i64>, tensor<i64>) -> tensor<4x4xf32>`

Reply:
@eric-k256 @silvasean - Rebased and updated the CustomOp with `config` attribute set to "torch_mlir".
@AmosLewis - Done, thanks.